addons/website_sale/static/src/scss/website_sale.scss

```diff
@@ -221,6 +200,8 @@ $input-border-color: $gray-400;
     .oe_product_image {
         min-width: var(--o-wsale-card-thumb-size);
         width: var(--o-wsale-card-thumb-size);
+        box-shadow: var(--o-wsale-card-thumb-shadow);
```

Comment:
Defining the shadow here would create an issue when the Style is defined to Thumbnails and the Layout to List :
![Capture d’écran 2024-08-16 à 13 02 35](https://github.com/user-attachments/assets/62bef150-be5f-4672-975b-38088a6b2ffb)

Reply:
I can't replicate (neither locally or on runbot)🤔
Could you provide me the necessary steps? 

<img width="719" alt="image" src="https://github.com/user-attachments/assets/2874b8c1-0168-473e-aedd-0c5e5f9303bb">

Reply:
It happens when I start playing with the Image Size options and switching from Grid to List 🤔 : 
https://www.awesomescreenshot.com/video/30632746?key=3bd5e595b2c7efb05dbc34c8a338fe60

Reply:
Ok, I see, thanks. It was a little glitch, fixed 👍